src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -25,6 +27,11 @@ const formatCourseRunTitle = (courseRun) => {
     return (
       <div className="course-run-label">
         <span>{`Course run starting on ${labelItems.join(' - ')}`}</span>
+        {/*
+          After we have a way of determining if the course run has been edited, that should be
+          added into the list of statuses being passed into the Pill component.
+        */}
```

Comment:
nit: I'd toss a `TODO:` on the front of that - makes it a pop a little more - scanning this comment would make me think it's just normal dev comments about what the code is doing.